src/timestamp.js

```diff
@@ -7,6 +7,8 @@
 
 const GLib = imports.gi.GLib;
 const Gio = imports.gi.Gio;
+const Gettext = imports.gettext;
+window._ = Gettext.gettext;
```

Comment:
We should move that initialization to an earlier place so it is accessible to all the places, see https://github.com/endlessm/eos-social/blob/master/EosSocial/main.js#L11 as an example.